docs/monetization-event-asset-code.md

```diff
@@ -0,0 +1,36 @@
+---
+id: monetization-event-asset-code
+title: MonetizationEvent.assetCode
+sidebar_label: MonetizationEvent.assetCode
+---
+
+import Specifications from '@site/src/components/Specifications';
+import BrowserCompat from '@site/src/components/BrowserCompat';
+
+The **`assetCode`** property of the [MonetizationEvent](monetization-event.md) interface returns the currency code.
+
+## Value
+
+A string representing the three-character currency code according to the last receipt.
```

Comment:
Again, if we want to deprecate receipts, I wouldn't want to mention them that often.